iam.tf

```diff
@@ -21,6 +21,7 @@ EOF
 }
 
 resource "aws_iam_policy" "int_tableau" {
+  name = "${var.application_name}-ssm-parameter"
```

Comment:
why the ssm-parameter?

Reply:
becuase this policy is soley dedicated to ssm parameters line 23- 92